ceph/ceph_admin/bootstrap.py

```diff
@@ -345,6 +345,8 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
                 "prometheus",
                 "node_exporter",
                 "alertmanager",
+                "promtail",
```

Comment:
@manasagowri Let's add `LOKI` service also over here, bcz this is also one of the monitoring stack images get used update in future releases, so at that to override we can just pass that custom images in metadata yaml files.

Reply:
Done.